docs/src/config.yaml

```diff
@@ -2,7 +2,7 @@
 keep_alive: 75
 
 # The socket address to bind [default: 0.0.0.0:3000]
-listen_addresses: '0.0.0.0:3000'
+listen_addresses: "0.0.0.0:3000"
```

Comment:
Sorry, my formatter changed these to double quotes. If this is a bother, LMK and I'll revert.